docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "WebLogic Domain"
+date: 2019-10-01T14:32:31-05:00
+weight: 2
+description: "WebLogic domains and operator's logs"
+---
+
+
+#### Overview
+This page describes to how to configure a WebLogic domain to use fluentd to send log information to Elasticsearch.
+
+The general mechanism of how this works is:
```

Comment:
The general mechanism of how this works is: -> Here's the general mechanism for how this works: